FRENCH/src/ch08-03-hash-maps.md

```diff
@@ -591,20 +589,20 @@ lesquels vous êtes maintenant en mesure de résoudre :
   alphabetically.
 -->
 
-* A partir d'une liste d'entier, utiliser un vecteur et retourner la moyenne, la
-  médiane (la valeur au milieu lorsque la liste est triée), et la valeur modale
-  (la valeur qui apparaît le plus souvent ; une table de hachage sera utile dans
-  ce cas) de la liste.
-* Convertir des chaînes de caractères en Louchébem. La première syllabe de
-  chaque mot est déplacée à la fin du mot et un suffixe argotique et substituée
-  par un `l` (L minuscule), ainsi “bonjour” devient “*l*onjour*bem*”. Les mots
-  qui commencent par un L ou une voyelle remplacent la syllabe suivante par un L
-  et rétablit en fin de mot la consonne initiale avec un suffixe libre. Et
-  gardez en tête les détails à propos de l'encodage UTF-8 !
+* À partir d'une liste d'entiers, utiliser un vecteur et retourner la moyenne,
+  la médiane (la valeur au milieu lorsque la liste est triée), et le mode (la
+  valeur qui apparaît le plus souvent ; une table de hachage sera utile dans ce
+  cas) de la liste.
+* Convertir des chaînes de caractères dans une variante du louchébem.
+  La consonne initiale de chaque mot est remplacée par la lettre `l` et est
+  rétablie à la fin du mot suivie du suffixe argotique “em” ; ainsi, “bonjour”
+  devient “*l*onjour*bem*”. Si le mot commence par une voyelle, ajouter un `l`
+  au début du mot et ajouter à la fin le suffixe “muche”. Et gardez en tête les
```

Comment:
```suggestion
  au début du mot et ajouter à la fin un suffixe libre. Et gardez en tête les
```

Looks like this is more complex than simply add `muche` at the end ?
Some examples in [Wikipedia](https://fr.wikipedia.org/wiki/Louch%C3%A9bem#Le_louch%C3%A9bem_des_bouchers_aujourd'hui) show that it is not always `muche` when it starts with a vowel ?

Reply:
Real louchébem is indeed more complex because you can basically choose any slang suffix you want (except for words that have agreed-upon forms, such as *douce* → *loucedé* or *fou* → *loufoque*), and transformation of words starting a vowel is even more complex.
In our case, I think it is best to present a simplified variant of louchébem with predefined suffixes, so that the exercise is simpler and less ambiguous; it would be too difficult for the reader to convert strings to real louchébem (at the very least, much more difficult than converting to pig latin).

Reply:
Yes, I agree with you, I understand better your change 😊
We don't need something very conservative with this very complex rules, it's just a light exercise 😉